src/Microsoft.DotNet.Cli.Utils/DotnetExec.cs

```diff
@@ -0,0 +1,54 @@
+using System.IO;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public static class DotnetExec
+    {
+        internal static string _hostDir;
+        internal static string _hostExePath;
+
+        /// <summary>
+        /// Gets the path to the version of corehost that was shipped with this command
+        /// </summary>
+        public static string LocalHostExePath => Path.Combine(PlatformServices.Default.Application.ApplicationBasePath, Constants.DotnetHostExecutableName);
```

Comment:
Copy Pasta :sweat_smile: 

Will make the change.

This file is a strawman for how we could find dotnet.exe after shared framework is installed. I expect it may undergo significant changes as that shapes up.